lte/gateway/c/core/oai/tasks/nas/emm/emm_data_ctx.cpp

```diff
@@ -48,7 +48,7 @@ extern "C" {
 #include "lte/gateway/c/core/oai/lib/3gpp/3gpp_33.401.h"
 #include "lte/gateway/c/core/oai/lib/3gpp/3gpp_36.401.h"
 #include "lte/gateway/c/core/oai/lib/secu/secu_defs.h"
-#include "lte/gateway/c/core/oai/tasks/mme_app/mme_app_timer.h"
+#include "lte/gateway/c/core/oai/tasks/mme_app/mme_app_timer.hpp"
```

Comment:
In this file, emm_ctx_clear_ue_nw_cap() is defined under extern C. Now extern C can be removed

Reply:
This is not under extern, it is temporarily moved to emm_headers.hpp file as some functions were throwing undefined reference even though they are declared/defined and used in hpp/cpp files. emm_headers.hpp file will be removed after all the files are moved to c++

Reply:
Could you add comment with mentioning github issue id i.e #13096